shared/health/src/test/java/io/pravega/shared/health/StatusAggregatorTests.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.health;
+
+import io.pravega.shared.health.impl.StatusAggregatorImpl;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class StatusAggregatorTests {
+
+    List<Status> statuses = new ArrayList<>(Arrays.asList(Status.UP, Status.UP, Status.DOWN));
```

Comment:
private static final

Also, why are you making a copy of `Arrays.asList`???

Reply:
Lol. Brain-fart.